### PR TITLE
Add funding acknowledgement

### PR DESCRIPTION
**Pull request by dladd**
Adds funding acknowledgement to libCellML ReadTheDocs documentation.

Addresses #90

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -39,3 +39,15 @@ Indices and tables
 * :ref:`modindex`
 * :ref:`search`
 
+Supported by:
+------------
+
+.. figure:: docs/images/vpr.png
```
Comment (nickerso) [inline docs/index.rst]: should just be `images/vpr.png` I think.

**Comment by nickerso** [comment]
All looks good to me.

**Comment by agarny** [comment]
The rendering looks a bit strange to me:
image

Regarding the VPR logo, there is thin border to the left and that cyan border at the top, which don't look great. As for the ABI logo, I wonder why there is that dotted line at the bottom...?!

**Comment by dladd** [comment]
Thanks @agarny - I've fixed the border issue on the vpr logo ([new version here](http://autotest.bioeng.auckland.ac.nz/libcellml-buildbot/fc2fc9d55fc6aeb9408234a5a766271b98e26fb1/dox/index.html)). I believe the dotted lines are the style for hyperlinks- you'll see they go solid when you mouseover the logo. I could look into adding an exception for images (I assume there is a stylesheet somewhere in our theme) but it just depends how much time we want to spend on this.

**Comment by nickerso** [comment]
The dotted line is fine. No need to tune this any more at this stage :)

**Pull request merged**